internal/util/util.go

```diff
@@ -67,9 +67,9 @@ func RoundOffCephFSVolSize(bytes int64) int64 {
 		return 4 * helpers.MiB
 	}
 
-	bytes /= helpers.MiB
+	floatbytes := float64(bytes) / helpers.MiB
```

Comment:
bytesInFloat sounds better? as @nixpanic suggested please add a E2E test for this one.

Reply:
Done.